ticdc/ticdc-changefeed-config.md

```diff
@@ -122,32 +122,36 @@ write-key-threshold = 0
-dispatchers = [
-    {matcher = ['test1.*', 'test2.*'], topic = "Topic expression 1", partition = "ts" },
-    {matcher = ['test3.*', 'test4.*'], topic = "Topic expression 2", partition = "index-value" },
-    {matcher = ['test1.*', 'test5.*'], topic = "Topic expression 3", partition = "table"},
-    {matcher = ['test6.*'], partition = "ts"}
-]
+# dispatchers = [
+#     {matcher = ['test1.*', 'test2.*'], topic = "Topic expression 1", partition = "ts" },
+#     {matcher = ['test3.*', 'test4.*'], topic = "Topic expression 2", partition = "index-value" },
+#     {matcher = ['test1.*', 'test5.*'], topic = "Topic expression 3", partition = "table"},
+#     {matcher = ['test6.*'], partition = "ts"}
+# ]
 
 # The protocol configuration item specifies the protocol format of the messages sent to the downstream.
 # When the downstream is Kafka, the protocol can only be canal-json or avro.
 # When the downstream is a storage service, the protocol can only be canal-json or csv.
 # Note: This configuration item only takes effect if the downstream is Kafka or a storage service.
-protocol = "canal-json"
+# protocol = "canal-json"
 
 # The following three configuration items are only used when you replicate data to storage sinks and can be ignored when replicating data to MQ or MySQL sinks.
 # Row terminator, used for separating two data change events. The default value is an empty string, which means "\r\n" is used.
-terminator = ''
-# Date separator type used in the file directory. Value options are `none`, `year`, `month`, and `day`. `none` is the default value and means that the date is not separated. For more information, see <https://docs.pingcap.com/tidb/dev/ticdc-sink-to-cloud-storage#data-change-records>. 
+# terminator = ''
+# Date separator type used in the file directory. Value options are `none`, `year`, `month`, and `day`. `none` is the default value and means that the date is not separated. For more information, see <https://docs.pingcap.com/tidb/dev/ticdc-sink-to-cloud-storage#data-change-records>.
 # Note: This configuration item only takes effect if the downstream is a storage service.
 date-separator = 'none'
+<<<<<<< HEAD
 
 # Whether to use partitions as the separation string. The default value is true, which means that partitions in a table are stored in separate directories. For usage examples, see <https://docs.pingcap.com/tidb/dev/ticdc-sink-to-cloud-storage#data-change-records)>. 
+=======
+# Whether to use partitions as the separation string. The default value is true, which means that partitions in a table are stored in separate directories. It is recommended that you keep the value as `true` to avoid potential data loss in downstream partitioned tables <https://github.com/pingcap/tiflow/issues/8724>. For usage examples, see <https://docs.pingcap.com/tidb/dev/ticdc-sink-to-cloud-storage#data-change-records)>.
+>>>>>>> 3948693f3 (ticdc: Fix some ticdc docs master 6.5 (#13664))
```

Comment:
```suggestion

# Whether to use partitions as the separation string. The default value is true, which means that partitions in a table are stored in separate directories. It is recommended that you keep the value as `true` to avoid potential data loss in downstream partitioned tables <https://github.com/pingcap/tiflow/issues/8724>. For usage examples, see <https://docs.pingcap.com/tidb/dev/ticdc-sink-to-cloud-storage#data-change-records)>.
```